crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -885,6 +986,13 @@ fn add_external_device_registration_message(
     false
 }
 
+fn external_device_registration_message(entity: &EntityMetadata) -> Option<Message> {
```

Comment:
Could be moved to `entity_store.rs`

Reply:
No, looking closer, this method is c8y related. The point is to register the device on the cloud.

Reply:
The function that should be moved to `entity_store.rs` is `auto_register_entity()`.

Reply:
> The function that should be moved to `entity_store.rs` is `auto_register_entity()`.

Done